### DISCUSSION TITLE
绳网应该没有脚本了吧？

### DISCUSSION THREAD

**Discussion by Jie-Ket** [share121/inter-knot#1878] [其他]
之前在其他网站发委托，总是有人发“这个委托我接了！”的信息，私聊后又说一堆有的没的，希望绳网的机器人识别能力一定要强啊！起码不要让这种机器人泛滥了！

**Comment by CloudSwordSage** [comment]
这个委托我接了\\dog
---
![image](https://github.com/user-attachments/assets/1eefbde0-3f34-4e19-9a79-8e6566e75a99)
